src/test/java/org/assertj/core/api/AbstractAssert_areEqual_Test.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2022 the original author or authors.
+ */
+package org.assertj.core.api;
```

Comment:
I suspect this test will pass even if `areEqual` is package private as both `TestAssert` and `AbstractAssert` are in org.assertj.core.api, let's move this test to another package

Reply:
Right, I've done the setup before moving the test class one level higher :facepalm: will clean it and enforce that the method stays protected

Reply:
I've taken a slightly different direction than moving the test class elsewhere. Please check it!